shogun-boot/application.yml

```diff
@@ -97,7 +99,7 @@ spring:
           timeout: 8000
 
 keycloak:
-  auth-server-url: https://${KEYCLOAK_HOST}:${KEYCLOAK_PORT}/auth
+  auth-server-url: https://${KEYCLOAK_HOST}/auth
```

Comment:
Is this intended? If so, the `KEYCLOAK_PORT` can be removed?

Reply:
Yes, I removed all references to it with b8d233b